packages/vite-plugin-cloudflare/src/cloudflare-environment.ts

```diff
@@ -119,6 +119,7 @@ const cloudflareBuiltInModules = [
 ];
 
 const defaultConditions = ["workerd", "module", "browser"];
+const target = "es2022";
```

Comment:
nit: `target` is quite generic, maybe something like `buildTarget` would be a bit clearer? 🤔 

(well.. you can say the same about `defaultConditions` 😅)